app/main/routes_frontend.py

```diff
@@ -418,40 +418,44 @@ def setup():
         elif 'interface' in payload:
             if payload['interface'] == 'wlan0':
                 # change wireless configuration (wpa_supplicant-wlan0.conf and wpa_supplicant.conf)
-                # sudo sed -i -e"s/^ssid=.*/ssid=\"$SSID\"/" /etc/wpa_supplicant/wpa_supplicant.conf
+                # sudo sed -i -e"s/^\bssid=.*/ssid=\"$SSID\"/" /etc/wpa_supplicant/wpa_supplicant.conf
```

Comment:
in regex '\b' marks a word boundary, so adding this will prevent this sed command (and later) from modifying a commented or commented out `bssid=<value>` entry